src/views/Community/index.tsx

```diff
@@ -182,7 +173,7 @@ const Community: React.FC<{ isLoading?: boolean; communityData: any; }> = (props
     };
 
     return (
-        <ViewContainer isLoading={loading || isLoading || refreshingPage || loadingManagers || loadingMerchants}>
+        <ViewContainer isLoading={loading || isLoading || refreshingPage || loadingManagers || loadingMerchants || loadingAmbassador || loadingPromoter || loadingContract}>
```

Comment:
WDYT of moving this validation to a variable outside the render - seems a lot "ors" to stay there 😄 ?